src/Commands/MachineToken/ListCommand.php

```diff
@@ -20,8 +22,24 @@ class ListCommand extends TerminusCommand
      * @usage terminus machine-token:list
      *   Lists your user's machine tokens
      */
-    public function listTokens() {
+    public function listTokens($options = ['format' => 'table', 'fields' => '']) {
```

Comment:
`format` and `fields` should be global options ala `--yes`. I can add that once #1179 is merged